spec/spec_helper.cr

```diff
@@ -3,296 +3,298 @@
 ENV["CRYSTAL_PATH"] = "#{__DIR__}/../src"
 
 require "spec"
-require "../src/compiler/crystal/**"
-require "./support/syntax"
-
-class Crystal::Program
-  def union_of(type1, type2, type3)
-    union_of([type1, type2, type3] of Type).not_nil!
+{% unless flag?(:win32) %}
```

Comment:
Why not just `skip_file`? It can skip parts of files you know :)

Reply:
Oooh, that's definitely better!